garden-backend-service/src/api/routes/_utils.py

```diff
@@ -44,3 +50,73 @@ async def is_doi_registered(doi: str) -> bool:
         return True
     else:
         return False
+
+
+def deprecated(
+    name="Endpoint",
+    message: str = None,
+    doc_url: str = "https://api-dev.thegardens.ai/docs",
```

Comment:
lets point this at the prod api endpoint by default 
```suggestion
    doc_url: str = "https://api.thegardens.ai/docs",
```